sdk/networkcloud/Azure.ResourceManager.NetworkCloud/README.md

```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure NetworkCloud management client library for .NET
+
+**[Describe the service briefly first.]**
```

Comment:
Please provide the service description here

Reply:
I added the description.